### PR TITLE
Allow stopping multiple apps

### LINKED ISSUE #107: feature request: stop all running apps
from: https://github.com/microsoft/vscode-spring-boot-dashboard/issues/28#issuecomment-638321659  by @brunovieira97 

I think adding a "stop" button should work. It's on title bar of the explorer, with confirmation poped up, in case of clicking by mistake. Is that enough, or any concern?

### PR DESCRIPTION
**Pull request by brunovieira97**
Created a new button to allow stopping multiple running apps, based on already existing code for starting apps.

Closes #107

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -168,6 +179,11 @@
                     "command": "spring-boot-dashboard.localapp.debug-multiple",
                     "when": "view == spring-boot-dashboard && !viewItem",
                     "group": "action@12"
+                },
+                {
+                    "command": "spring-boot-dashboard.localapp.stop-multiple",
+                    "when": "view == spring-boot-dashboard && !viewItem",
+                    "group": "action@13"
```
Comment (brunovieira97) [inline package.json:186]: I wasn't very sure about the `group` item, as it doesn't seem to be used anywhere.

Comment (Eskibear) [reply package.json:186]: > it doesn't seem to be used anywhere

It was contributing to context menu when you right click empty place inside this view. However it's not working now, which is probably caused by upstream changes in vscode.

**Comment by brunovieira97** [comment]
I actually tried that, but didn't find a way to do so. Since the Run button is also shown if all apps are running, I ended up leaving as is.

Will take a better look at it when I have the time ;)

File: package.json (hunk 1 above)
Comment (brunovieira97) [reply package.json:186]: Got it, thanks for explaining!

**Pull request merged**